File: AV1/NikitaLogisticsSA_Python/transports/ship.py
from transports.transport import Transport


class Ship(Transport):
    _cost: float = 0

    def __init__(self):
        self._cost = 9

    def deliver(self, distance: float) -> float:
        self.to_sail(distance)
        return distance * self._cost

    def to_sail(self, distance: float):
            print(f"Deliver by sea in a container. Distance {distance} Km.")

File: AV1/NikitaLogisticsSA/Ship.java
public class Ship implements Transport{

    private float cost;
    
    public Ship(){
        cost = 10;
    }

    @Override
    public float deliver(float distance){
        toSail(distance);
        return distance * cost;
    }

    private void toSail(float distance){
        System.out.println("Deliver by sea in a container. Distance: " + distance + " Km.");
    }
}